chaos-monkey-spring-boot/src/test/java/de/codecentric/spring/boot/chaos/monkey/component/ChaosMonkeyRequestScopeTest.java

```diff
@@ -35,6 +35,7 @@
 /**
  * @author Benjamin Wilms
  */
+// TODO migratie to @RunWith(MockitoExtension.class)
```

Comment:
`// TODO migrate to @ExtendWith(MockitoExtension.class)`

Typo in migrate and it should be ExtendWith instead of RunWith. Nothing serious, though :smiley:

Reply:
Good catch! :bowing_man: